core/src/test/java/org/infinispan/remoting/FailureType.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.remoting;
+
+import org.infinispan.protostream.annotations.ProtoEnumValue;
+import org.infinispan.protostream.annotations.ProtoName;
+
+@ProtoName("FailureType")
```

Comment:
```@ProtoName``` is not required unless we want to specify a new name. It also has the side effect of forcing this class to be processed in case the inner annotations are not detected. but that would be a bug. did you have to use force here?